packages/config/src/config/database.ts

```diff
@@ -28,7 +28,7 @@ const wait = (time: number) =>
 
 const connect = async (): Promise<void> => {
   try {
-    await mongoose.connect(MONGO_URL, { autoReconnect: true })
+    await mongoose.connect(MONGO_URL)
```

Comment:
was this autoReconnect not actually used by mongoose?

Reply:
In previous versions we had to enable it manually. But from version 6 onward, it's enabled by default and the option has been removed entirely